frontend/src/pages/rateLater/RateLater.tsx

```diff
@@ -127,11 +109,11 @@ const RateLaterPage = () => {
           can also help you import videos effortlessly.
           <br />
           You will then be able to rate the videos you imported.
-        </span>
-        <br />
-      </div>
-
-      <RateLaterAddForm addVideo={addToRateLater} />
+        </CardContent>
+        <CardActions>
+          <RateLaterAddForm addVideo={addToRateLater} />
+        </CardActions>
+      </Card>
```

Comment:
I'm not a fan of the style of this card because it looks different to other cards that we use on Tournesol (VideoCard or PeopleCard and Partner cards on about page). I see we have a lot of code duplication for the card's style. I didn't see it clearly before, but the card on the comparison page also have a different style. How about making a uniform custom Card component for Tournesol? Probably best done in a future PR.

![Screenshot 2021-12-19 at 08 40 36](https://user-images.githubusercontent.com/8818081/146668304-3e1062e8-03b5-4249-b82f-248c8e6820db.png)

Reply:
Yes, I'm not convinced myself about this card in this usecase.  
Perhaps a component similar to the search filters, or to the `Section` used in settings would make more sense.